src/lib/sql/sql_translator.cpp

```diff
@@ -191,20 +194,28 @@ std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_statement(const hsql:
 
 std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_select_statement(const hsql::SelectStatement& select) {
   // SQL Orders of Operations
-  // 1. FROM clause (incl. JOINs and sub-SELECTs that are part of this)
-  // 2. SELECT list (to retrieve aliases)
-  // 3. WHERE clause
-  // 4. GROUP BY clause
-  // 5. HAVING clause
-  // 6. SELECT clause (incl. DISTINCT)
-  // 7. UNION clause
-  // 8. ORDER BY clause
-  // 9. LIMIT clause
+  // 1. WITH clause
+  // 2. FROM clause (incl. JOINs and sub-SELECTs that are part of this)
+  // 3. SELECT list (to retrieve aliases)
+  // 4. WHERE clause
+  // 5. GROUP BY clause
+  // 6. HAVING clause
+  // 7. SELECT clause (incl. DISTINCT)
+  // 8. UNION clause
+  // 9. ORDER BY clause
+  // 10. LIMIT clause
 
   AssertInput(select.selectList, "SELECT list needs to exist");
   AssertInput(!select.selectList->empty(), "SELECT list needs to have entries");
   AssertInput(!select.unionSelect, "Set operations (UNION/INTERSECT/...) are not supported yet");
 
+  // Translate WITH clause
+  if (select.withDescriptions) {
+    for (const auto& desc : *select.withDescriptions) {
+      _translate_hsql_with_description(*desc);
```

Comment:
for consistent naming, I guess that should be called "_translate_hsql_with_description".

Since "desc" is a hyrise variable, it should have an unabbreviated name

Reply:
> for consistent naming, I guess that should be called "_translate_hsql_with_description".

Am I blind, I do not see any difference.

> Since "desc" is a hyrise variable, it should have an unabbreviated name

So `*with_description` instead of `*desc`?

Reply:
> I may be blind, but I do not see the difference.

Neither do I

> So *with_description instead of *desc?

Yes.